### ISSUE TITLE
Internet Explorer viser ikke alle tavler

### ISSUE THREAD

**Issue by michaelkrog** [henrikravn/symbol-mobile#88]
@henrikravn skriver:
> Ser ud til at eneste Explorer problem er visning af nogle tavler i kalender...enkelte vises?

**Comment by henrikravn** [comment]
- evt. fix til sidst - vi fremviser på chrome på messen.
Dog vigtig ved åbning.

**Comment by henrikravn** [comment]
mange kunder på skoler og kommuner bruger stadig explorer - og har ikke mulighed for at install. chrome el. firefox

**Comment by michaelkrog** [comment]
Hvilken version af IE er det du oplever problemet i?

**Comment by michaelkrog** [comment]
Bemærk følgende fra Microsoft:
>Beginning January 12, 2016, only the most current version of Internet Explorer available for a supported operating system will receive technical support and security updates. Please visit the Internet Explorer Support Lifecycle Policy FAQ here http://support.microsoft.com/gp/Microsoft-Internet-Explorer for list of supported operating systems and browser combinations.

IE versioner før 10 er allerede ikke supporteret længere. Om 10 måneder er det kun IE 11 der er supporteret.
http://support.microsoft.com/lifecycle/search/?sort=PN&alpha=internet+explorer

**Comment by henrikravn** [comment]
Er testet på nyeste version...der vises ikke forløb og nogle tavler vises ikke - hverken i rediger kalender og i kalender.
Kan ikke lige gennemskue om der er et system i det?

**Comment by michaelkrog** [comment]
Der er 2 problemer i forbindelse med IE. 
1. IE kan ikke, som andre browsere, finde ud af hvad størrelse en SVG-fil har, før den er forbundet til DOM'en. Derfor returnerer den blot 0x0 pixels og bliver dermed ikke vist. Jeg tvinger den til at være 350x350 pixels i det tilfælde hvilket hjælper lidt.
2. IE ser ud til at ignorere nogle jpg-filer i forbindelse med skalering. Dette skal undersøges nærmere.

**Comment by henrikravn** [comment]
At tvinge den til 350X350px har kun indflydelse på browser visning, ik?
Med venlig hilsen

Henrik Ravn

[cid:C315D14B-2FD6-448D-974E-B5BE6FEEE4C2]

9930 7583 | <her@exakt.dk> her<mailto:her@exakt.dk><mailto:lhk@exakt.dk><her@exakt.dk>@exakt.dk<mailto:her@exakt.dk>
Stenholm 7 | 9400 Nørresundby | 9819 3533 | exakt.dk<http://exakt.dk/shop/frontpage.html>

Find også Exakt på: Facebook<https://www.facebook.com/exaktdk> | LinkedIn<https://www.linkedin.com/company/exakt?trk=company_name> | Google+<https://plus.google.com/110183979779133868817/about> | Twitter<https://twitter.com/positiverne>

Se vores aktuelle aviser her<http://exakt.dk/shop/cms-23.html>

Fra: Michael Krog <notifications@github.com<mailto:notifications@github.com>>
Svar til: henrikravn/symbol-mobile <reply@reply.github.com<mailto:reply@reply.github.com>>
Dato: tirsdag den 17. marts 2015 10.17
Til: henrikravn/symbol-mobile <symbol-mobile@noreply.github.com<mailto:symbol-mobile@noreply.github.com>>
Cc: Henrik Ravn <her@exakt.dk<mailto:her@exakt.dk>>
Emne: Re: [symbol-mobile] 6. Internet Explorer viser ikke alle tavler (#88)


Der er 2 problemer i forbindelse med IE.
1. IE kan ikke, som andre browsere, finde ud af hvad størrelse en SVG-fil har, før den er forbundet til DOM'en. Derfor returnerer den blot 0x0 pixels og bliver dermed ikke vist. Jeg tvinger den til at være 350x350 pixels i det tilfælde hvilket hjælper lidt.
2. IE ser ud til at ignorere nogle jpg-filer i forbindelse med skalering. Dette skal undersøges nærmere.

—
Reply to this email directly or view it on GitHub<https://github.com/henrikravn/symbol-mobile/issues/88#issuecomment-82226171>.

________________________________
Denne mail er blevet scannet af http://www.comendo.com og indeholder ikke virus!
________________________________


********************************************************************************
Denne mail er blevet scannet af http://www.comendo.com og indeholder ikke virus!
********************************************************************************

**Comment by michaelkrog** [comment]
Jo. Det er kun visningen der tvinges. Det ændrer ikke på hvordan filen er gemt.

**Comment by henrikravn** [comment]
Og det ændre ikke oplevelsen på app'en på f.eks. en smartphone?
Med venlig hilsen

Henrik Ravn

[cid:60DB7DB9-7047-4D93-BAF8-AF38E4D488CC]

9930 7583 | <her@exakt.dk> her<mailto:her@exakt.dk><mailto:lhk@exakt.dk><her@exakt.dk>@exakt.dk<mailto:her@exakt.dk>
Stenholm 7 | 9400 Nørresundby | 9819 3533 | exakt.dk<http://exakt.dk/shop/frontpage.html>

Find også Exakt på: Facebook<https://www.facebook.com/exaktdk> | LinkedIn<https://www.linkedin.com/company/exakt?trk=company_name> | Google+<https://plus.google.com/110183979779133868817/about> | Twitter<https://twitter.com/positiverne>

Se vores aktuelle aviser her<http://exakt.dk/shop/cms-23.html>

Fra: Michael Krog <notifications@github.com<mailto:notifications@github.com>>
Svar til: henrikravn/symbol-mobile <reply@reply.github.com<mailto:reply@reply.github.com>>
Dato: tirsdag den 17. marts 2015 10.27
Til: henrikravn/symbol-mobile <symbol-mobile@noreply.github.com<mailto:symbol-mobile@noreply.github.com>>
Cc: Henrik Ravn <her@exakt.dk<mailto:her@exakt.dk>>
Emne: Re: [symbol-mobile] 6. Internet Explorer viser ikke alle tavler (#88)


Jo. Det er kun visningen der tvinges. Det ændrer ikke på hvordan filen er gemt.

—
Reply to this email directly or view it on GitHub<https://github.com/henrikravn/symbol-mobile/issues/88#issuecomment-82237491>.

________________________________
Denne mail er blevet scannet af http://www.comendo.com og indeholder ikke virus!
________________________________


********************************************************************************
Denne mail er blevet scannet af http://www.comendo.com og indeholder ikke virus!
********************************************************************************

**Comment by michaelkrog** [comment]
Nej. Det er kun hvis browseren ikke kan finde ud af hvort billedet er. Og det er kun IE der kæmper med det problem.

**Comment by michaelkrog** [comment]
Det ser ud til at det ikke kun er jpg-filer IE har problemer med, men også udvalgte png-filer. Det er dog ikke alle kun nogle af filerne. Selvom IE melder dem som færdigindlæst, er Image objektet tilsyneladende ikke korrekt.
